bitcoin/src/blockdata/transaction.rs

```diff
@@ -387,7 +387,7 @@ mod tmp {
         /// To use a custom value, use [`minimal_non_dust_custom`].
         ///
         /// [`minimal_non_dust_custom`]: TxOut::minimal_non_dust_custom
-        pub fn minimal_non_dust(script_pubkey: ScriptBuf) -> Self {
+        pub fn minimal_non_dust(script_pubkey: ScriptBuf) -> TxOut {
```

Comment:
Why is this needed? The extension trait is implemented for `TxOut` so `Self` refers to `TxOut` thus it shouldn't be needed.

Reply:
Oh I have no idea why I had it in mind that doing this was part of creating the extension traits. Removed.